app/javascript/packs/application.js

```diff
@@ -0,0 +1,18 @@
+/* eslint no-console:0 */
```

Comment:
Does this file clash with [`app/webpacker/packs/applications.js`](https://github.com/guy-roberts/eyfs-reform-spike/blob/401f6a7f2abb1690267d658bafcf0763b79bf337/app/webpacker/packs/application.js)?

Reply:
not sure why webpacker changed this file but i think we should stick with the original code in there and not commit this empty file so ill re-add those changes

Reply:
this is an autogenerated file by webpacker, that's all i know :-)